README.md

```diff
@@ -81,10 +81,13 @@ OPTIONS
     -m          Memory Size (MB)    (default: 1024)
     -M          Mac address         (default: auto-assigned)
     -p          Console port        (default: auto)
+    -P          User password       (default: false)
```

Comment:
I think this option gives me the impression that I'll be able to set the password for the user but that's not the point, right?

Reply:
This option accepts a boolean. If it's true, you are asked to set a user password in the command  prompt after you run the command.

Reply:
I changed it to "Enable User password" to make it less ambigious